Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/deletionwarning/SubmissionDeletionWarningFragment.kt

```diff
@@ -79,10 +80,17 @@ class SubmissionDeletionWarningFragment : Fragment(R.layout.fragment_submission_
                     popBackStack()
                 }
                 is State.TestRegistered -> when {
-                    state.test.isPositive ->
-                        NavGraphDirections.actionToSubmissionTestResultAvailableFragment(testType = state.test.type)
-                            .run { findNavController().navigate(this, navOptions) }
-
+                    state.test.isPositive -> {
```

Comment:
You can put the .run{} after the else block, so you dont have to use it twice. Also I would change the _run_ to an _also_

Reply:
updated